Library/Styles/CheckoutStyles.swift

```diff
@@ -0,0 +1,50 @@
+import Prelude
+import UIKit
+
+public func checkoutAdaptableStackViewStyle(_ isAccessibilityCategory: Bool) -> (StackViewStyle) {
+  return { (stackView: UIStackView) in
+    let alignment: UIStackView.Alignment = (isAccessibilityCategory ? .leading : .center)
+    let axis: NSLayoutConstraint.Axis = (isAccessibilityCategory ? .vertical : .horizontal)
+    let distribution: UIStackView.Distribution = (isAccessibilityCategory ? .equalSpacing : .fill)
+    let spacing: CGFloat = (isAccessibilityCategory ? Styles.grid(1) : 0)
+
+    return stackView
+      |> \.alignment .~ alignment
+      |> \.axis .~ axis
+      |> \.distribution .~ distribution
+      |> \.spacing .~ spacing
+  }
+}
+
+public let checkoutBackgroundStyle: ViewStyle = { (view: UIView) in
+  view
+    |> \.backgroundColor .~ UIColor.ksr_grey_300
+}
+
+public let checkoutRoundedCornersStyle: ViewStyle = { (view: UIView) in
+  view
+    |> \.layer.cornerRadius .~ 6
+}
+
+public let checkoutStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
+  stackView
+    |> \.axis .~ NSLayoutConstraint.Axis.vertical
+    |> \.isLayoutMarginsRelativeArrangement .~ true
+    |> \.layoutMargins .~ UIEdgeInsets(
+      top: Styles.grid(2), left: Styles.grid(4), bottom: Styles.grid(3), right: Styles.grid(4)
+    )
+    |> \.spacing .~ (Styles.grid(1) + Styles.gridHalf(1))
+}
+
+public let checkoutTitleLabelStyle: LabelStyle = { (label: UILabel) in
+  label
+    |> \.accessibilityTraits .~ UIAccessibilityTraits.header
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.font .~ UIFont.ksr_headline()
```

Comment:
Looking at the abstract files, I think the title labels are meant to have a font size of 15pts.

Reply:
(the default for `ksr_headline()` is 17pts)

Reply:
Good eye! Thanks for pointing this out.